recipe/build.sh

```diff
@@ -1,18 +1,15 @@
 #!/bin/bash
 
-if [[ $(uname) == Linux ]]; then
-  # FIXME: This is a terrible workaround.
-  # Ideally we should fix the information the .la files.
-  rm -rf $PREFIX/lib/libquadmath.la
-  rm -rf $PREFIX/lib/libgfortran.la
```

Comment:
@ocefpaf - I'm not sure what this was supposed to work around, but it's working now without this. Do you remember what this was for?